### PR TITLE
Adding Humata case study

### PR DESCRIPTION
**Pull request by craigcannon**

### CODE REVIEW

File: apps/www/_customers/humata.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      label: 'Solutions',
+      text: 'Supabase Database, Supabase Auth, Supabase Realtime, Supabase Vector',
+    },
+  ]
+about: Use AI to ask questions across all of your files.
+---
+
+## Introduction
+
+Humata, a platform enabling millions of users to chat with and analyse their all documents, relies on cutting-edge technology to support rapid growth and high user demand. Humata started as PDF AI that allows anyone to chat with their files, initially geared for scientific researchers. Humata has now expanded to serve major enterprises and government institutions by connecting their knowledge base to AI and allowing them to get answers they can trust with best-in-class referencing.  With a tech stack that includes Supabase, GCP, GPT-4o, Vercel, and NEXT.js, Humata’s platform provides real-time, mission-critical analysis for researchers, enterprises, and government institutions. By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.
```
Comment (dshukertjr) [inline apps/www/_customers/humata.mdx:34]: Not a blocker, but I feel that there is a good chance referencing the model name `GPT-4o` here would be outdated really soon. Maybe just saying `OpenAI` or something generic might be better in the long run? I'm okay keeping it the current way though, so your call!

**Review by dshukertjr — APPROVED** [review]
Other than the prettier issue, it looks great!

File: apps/www/_customers/humata.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      label: 'Solutions',
+      text: 'Supabase Database, Supabase Auth, Supabase Realtime, Supabase Vector',
+    },
+  ]
+about: Use AI to ask questions across all of your files.
+---
+
+## Introduction
+
+Humata, a platform enabling millions of users to chat with and analyse their all documents, relies on cutting-edge technology to support rapid growth and high user demand. Humata started as PDF AI that allows anyone to chat with their files, initially geared for scientific researchers. Humata has now expanded to serve major enterprises and government institutions by connecting their knowledge base to AI and allowing them to get answers they can trust with best-in-class referencing. With a tech stack that includes Supabase, GCP, GPT-4o, Vercel, and NEXT.js, Humata’s platform provides real-time, mission-critical analysis for researchers, enterprises, and government institutions. By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.
```
Comment (awalias) [inline apps/www/_customers/humata.mdx:34]:
```suggestion
Humata, a platform enabling millions of users to chat with and analyse their all documents, relies on cutting-edge technology to support rapid growth and high user demand. Humata started as PDF AI that allows anyone to chat with their files, initially geared for scientific researchers. Humata has now expanded to serve major enterprises and government institutions by connecting their knowledge base to AI and allowing them to get answers they can trust with best-in-class referencing. With a tech stack that includes Supabase, GCP, OpenAI, Vercel, and NEXT.js, Humata’s platform provides real-time, mission-critical analysis for researchers, enterprises, and government institutions. By partnering with Supabase, Humata achieved a 4X reduction in vector database costs, streamlined their development, and consolidated their data so they could scale seamlessly.
```

**Review by awalias — APPROVED** [review]

**Comment by awalias** [comment]
```
www:build: YAMLException: incomplete explicit mapping pair; a key node is missed; or followed by a non-tabulated empty line at line 3, column 35:
www:build:     title: Humata Scales with Supabase: Achieving 4X Cost Savings and  ... 
www:build:                                       ^
www:build:     at generateError (/vercel/path0/node_modules/js-yaml/lib/js-yaml/loader.js:167:10)
```

looks like turbo not happy with the`:`

File: apps/www/_customers/humata.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+name: Humata
+title: Humata Scales with Supabase: Achieving 4X Cost Savings and Enhanced Performance
```
Comment (awalias) [inline apps/www/_customers/humata.mdx:3]:
```suggestion
title: Humata Scales with Supabase - Achieving 4X Cost Savings and Enhanced Performance
```

File: apps/www/_customers/humata.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+name: Humata
+title: Humata Scales with Supabase - Achieving 4X Cost Savings and Enhanced Performance
+# Use meta_title to add a custom meta title. Otherwise it defaults to '{name} | Supabase Customer Stories':
+meta_title: Humata Scales with Supabase: Achieving 4X Cost Savings and Enhanced Performance
```
Comment (awalias) [inline apps/www/_customers/humata.mdx:5]:
```suggestion
meta_title: Humata Scales with Supabase - Achieving 4X Cost Savings and Enhanced Performance
```

**Pull request merged**